lib/tasks/token/destroy_expired_tokens.rake

```diff
@@ -0,0 +1,18 @@
+namespace :token do
```

Comment:
**Code Optimisation - Critical**
Add rake task to delete expired tokens. This works to optimise the logout feature.